src/coreclr/src/jit/liveness.cpp

```diff
@@ -2399,49 +2399,35 @@ void Compiler::fgInterBlockLocalVarLiveness()
         return;
     }
 
-    /*-------------------------------------------------------------------------
-     * Variables involved in exception-handlers and finally blocks need
-     * to be specially marked
-     */
+    //-------------------------------------------------------------------------
+    // Variables involved in exception-handlers and finally blocks need
+    // to be specially marked
+    //
     BasicBlock* block;
 
     VARSET_TP exceptVars(VarSetOps::MakeEmpty(this));  // vars live on entry to a handler
     VARSET_TP finallyVars(VarSetOps::MakeEmpty(this)); // vars live on exit of a 'finally' block
-    VARSET_TP filterVars(VarSetOps::MakeEmpty(this));  // vars live on exit from a 'filter'
```

Comment:
`filterVars` was never separately used (it was checking for membership of either set), so it is now combined with `exceptVars`. `finallyVars` is a subset of `exceptVars` and must be maintained to set `lvMustInit`